core/src/test/java/com/amazon/opendistroforelasticsearch/sql/expression/operator/predicate/UnaryPredicateOperatorTest.java

```diff
@@ -59,18 +65,37 @@ public void test_not_missing() {
   }
 
   @Test
-  public void is_null_predicate() {
-    FunctionExpression expression = dsl.isnull(DSL.literal(1));
+  public void test_is_null_predicate() {
+    FunctionExpression expression = dsl.is_null(DSL.literal(1));
     assertEquals(BOOLEAN, expression.type());
     assertEquals(LITERAL_FALSE, expression.valueOf(valueEnv()));
 
-    expression = dsl.isnull(DSL.literal(ExprNullValue.of()));
+    expression = dsl.is_null(DSL.literal(ExprNullValue.of()));
     assertEquals(BOOLEAN, expression.type());
     assertEquals(LITERAL_TRUE, expression.valueOf(valueEnv()));
   }
 
   @Test
-  public void is_not_null_predicate() {
+  public void test_isnull_predicate() {
+    ArrayList<Expression> exprValueArrayList = new ArrayList<>();
+    exprValueArrayList.add(dsl.literal("test"));
+    exprValueArrayList.add(dsl.literal(100));
+    exprValueArrayList.add(dsl.literal(""));
```

Comment:
np: you can refactor this in your PR by JUnit's parameterized test. Example: https://github.com/opendistro-for-elasticsearch/sql/blob/develop/core/src/test/java/com/amazon/opendistroforelasticsearch/sql/expression/function/WideningTypeRuleTest.java#L103

Reply:
Done,thx